WordPress/Classes/Categories/NSURLRequest+Helpers.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+
+@interface NSURLRequest (Helpers)
```

Comment:
Perhaps we could rename the getter to.. `requestForWordPressAuthentication`?. 

I'm not comfortable with the idea of subclassing NSURLRequest, mostly because we wouldn't be adding new attributes to it. This new class would contain a couple static getters (that could return just a NSURLRequest instance).